server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,70 @@
+const express = require("express");
```

Comment:
Did you add this in by accident ?

Reply:
I don't even know how this got here. I don't remember changing any backend files....
Maybe I merged dev into this branch before creating the pull request?